docs/src/components/ProviderMarquee.js

```diff
@@ -1,29 +1,30 @@
-import React from "react"
```

Comment:
I see you added a Shopify logo here, which is fine, but all the `'` characters need to be changed back to `"`